src/OrchardCore.Modules/Orchard.Alias/Views/_ViewImports.cshtml

```diff
@@ -1,4 +1,4 @@
-﻿@*
+@*
     For more information on enabling MVC for empty projects, visit http://go.microsoft.com/fwlink/?LinkID=397860
 
 *@
```

Comment:
Sometimes the editors change CR/LF  by CR or remove a space. This is expected when we open a file that hasn't been normalized yet.